commands_add.go

```diff
@@ -46,10 +46,31 @@ func (a addCmd) ToAuditEntry() string {
 		<username>%s</username>
 		<funds>%.02f</funds>
 	</userCommand>`,
-		time.Now().UnixNano()/1e6, redisBaseKey, a.id, a.userID, a.amount.ToFloat(),
+		time.Now().UnixNano()/1e6, redisBaseKey, a.txID, a.userID, a.amount.ToFloat(),
 	)
 }
 
 func (a addCmd) Execute() {
-	consoleLog.Warning("Not implemented: ADD")
+	// Create an account if one does not exist
+	if !accountStore.HasAccount(a.userID) {
+		consoleLog.Noticef("Creating account for %s", a.userID)
+		if err := accountStore.CreateAccount(a.userID); err != nil {
+			consoleLog.Error(err.Error())
```

Comment:
This is unnecessary since `abortTx` will (eventually) log the message to the console [here](https://github.com/DistributedDesigns/worker/blob/master/txWorker.go#L39)